### PR TITLE
XW-842 | Make it run under node 5.X

### PR DESCRIPTION
**Pull request by hakubo**
## Links
- https://nodejs.org/en/download/releases/
- https://github.com/kenkouot/hapi-i18next
## Description

We'd like to upgrade node to latest version. Currently 4.X is LTS I think we'd gain a lot if we jumped straight to 5.x. Then when 6 lands (next LTS) we could stay longer on it.

I ran it with:
`node v5.7.1
npm 3.8.1`

This branch just go as far as - make it run. There are still missing pieces that have to be solved:
- [ ] Fork hapi-18next under Wikia and install dev dependencies as dev dependencies
- [ ] test if all works as expected
- [ ] load test it

Branch made so when we have a way to deploy mercury with latest version - we have something to deploy.
## Reviewers

@rogatty @RafLeszczynski 
@rmarchei - when we have a way to deploy mercury with latest node - we can use this branch to test it
## Update

We didn't make it work with with node 5.7 but we made it work with node 5.10.1

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -78,6 +78,7 @@
     "multipipe": "0.1.2",
     "nodemon": "1.8.1",
     "npm-run-all": "1.4.0",
+    "phantomjs-prebuilt": "^2.1.4",
```
Comment (tomasznapieralski) [inline package.json]: use fixed version

File: server/gulpfile.js [hunk 2]
```diff
@@ -43,7 +42,7 @@ gulp.task('build-server-scripts', ['build-server-init-config'], function (done)
  * Copy node dependencies to www/server/
  */
 gulp.task('build-server-node-modules', function () {
-	var deps = '/{' + nodeDeps.join('/**/*,') + '/**/*}';
```
Comment (hakubo) [inline server/gulpfile.js]: npm 3 installs all as flat as possible so this won't work
unless we are more clever here - IMHO not worth it
node_modules in server take 37Megs

File: server/gulpfile.js [hunk 3]
```diff
@@ -43,7 +42,7 @@ gulp.task('build-server-scripts', ['build-server-init-config'], function (done)
...
+	var deps = '/**/*';
```
Comment (tomasznapieralski) [inline server/gulpfile.js]: everything everywhere, are you sure?

**Comment by hakubo** [comment]
latest node is v5.11.0 can we use that already?

**Comment by tomasznapieralski** [comment]
sure, we had problems with `apt-get` on Friday, but it seems that it is availabe now

**Comment by tomasznapieralski** [comment]
Blocked by https://wikia-inc.atlassian.net/browse/XW-1441

**Pull request closed without merging**